server/src/main/java/io/spine/examples/chatspn/server/message/MessageAggregate.java

```diff
@@ -64,4 +67,36 @@ private void event(MessagePosted e) {
                  .setContent(e.getContent())
                  .setWhenPosted(e.getWhenPosted());
     }
+
+    /**
+     * Handles the command to update a message content.
+     *
+     * @throws MessageContentCannotBeUpdated
+     *         if the message does not exist, or editor is not an original sender of this message
+     */
+    @Assign
+    MessageContentUpdated handle(UpdateMessageContent c) throws MessageContentCannotBeUpdated {
```

Comment:
Please check if this line fits 100 symbols wide. I am reviewing this PR away from my IDE, so I cannot tell, unfortunately.

Reply:
It fits, 96 symbols.